modules/nw-sriov-dpdk-example-intel.adoc

```diff
@@ -66,17 +66,20 @@ metadata:
   namespace: openshift-sriov-network-operator
 spec:
   networkNamespace: <target_namespace>
-  ipam: "{}" <1>
+  ipam: |-
+# ... <1>
   vlan: <vlan>
   resourceName: intelnics
 ----
-<1> Specify an empty object `"{}"` for the ipam CNI plug-in. DPDK works in userspace mode and does not require an IP address.
+<1> Specify a configuration object for the ipam CNI plug-in as a YAML block scalar. The plug-in manages IP address assignment for the attachment definition.
 +
 [NOTE]
 =====
 See the `Configuring SR-IOV additional network` section for a detailed explanation on each option in `SriovNetwork`.
 =====
 +
+An optional library, app-netutil, provides several API methods for gathering network information about a pod from within a container running within that pod.
```

Comment:
Please add the following link for reference: https://docs.openshift.com/container-platform/4.8/networking/hardware_networks/about-sriov.html#nw-sriov-app-netutil_about-sriov

Reply:
@EinatGlottmann, so we have some unusual limitations in our tooling in 2021; We can't link to other pages from within any `modules/` files. I can add an _Additional resources_ section for this, though, in `networking/hardware_networks/using-dpdk-and-rdma.adoc`.